spec/lib/gitlab/closing_issue_extractor_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'spec_helper'
+
+describe Gitlab::ClosingIssueExtractor do
+  let(:project) { create(:project) }
+  let(:issue) { create(:issue, project: project) }
+
+  before do
+    @iid1 = issue.iid
+  end
+
+  describe :closed_by_message_in_project do
+    context 'with a single reference' do
+      it { subject.closed_by_message_in_project("Awesome commit (Closes ##{@iid1})", project).should == [issue] }
```

Comment:
Please change these to `it do` and insert line breaks to bring this under 80 chars per line. Do the same for all the hound warnings.